test/com/google/javascript/jscomp/InlineAndCollapsePropertiesTest.java

```diff
@@ -47,7 +47,12 @@ protected CompilerPass getProcessor(final Compiler compiler) {
     return new CompilerPass() {
       AggressiveInlineAliases aggressiveInlineAliases = new AggressiveInlineAliases(compiler);
       CollapseProperties collapseProperties =
-          new CollapseProperties(compiler, PropertyCollapseLevel.ALL);
+          new CollapseProperties(
+              compiler,
+              PropertyCollapseLevel.ALL,
+              compiler.getOptions().chunkOutputType,
```

Comment:
Can we just hard-code these option to the simple case here and below (e.g. ChunkOutputType.GLOBAL_NAMESPACE, ResolutionMode.BROWSER, etc.)?

I think that makes it a bit simpler to follow what the behavior being tested actually is...

Reply:
done